tests/integration/test_benchmark.py

```diff
@@ -49,6 +49,21 @@ def test_benchmark_single_table_basic_synthsizers():
     ] == quality_scores.index.tolist()
 
 
+def test_benchmark_single_table_realtabformer_no_metrics():
+    """Test it without metrics."""
+    # Run
+    output = sdgym.benchmark_single_table(
+        synthesizers=['RealTabFormerSynthesizer'],
+        sdv_datasets=['student_placements'],
+        sdmetrics=[],
+    )
+
+    # Assert
+    assert set(output['Synthesizer']) == {'RealTabFormerSynthesizer'}
+    assert 'Train_Time' in output
+    assert 'Sample_Time' in output
```

Comment:
Can we assert that this actually has numerical values or that the process terminated with success ?